lib/routes/token.js

```diff
@@ -249,6 +249,18 @@ function confirmRefreshToken(params) {
       throw AppError.invalidScopes();
     }
     tokObj.scope = params.scope;
+
+    var now = new Date();
+    var lastUsedAt = tokObj.lastUsedAt;
+    var minimum_LastUsedAt_Delay_MS = 60000; // one minute. TODO: make it 24h
+
+    if ((lastUsedAt - now) > minimum_LastUsedAt_Delay_MS){
```

Comment:
for the `if` here we can `log` both that token was not updated or not updated (because it was already recently updated)